src/App.test.ts

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
You can do a quick Typescript check of your project with `npx tsc`

And right now the check is failing in this file